apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/main/kotlin/ch/tutteli/atrium/api/fluent/en_GB/expectExtensions.kt

```diff
@@ -15,19 +16,37 @@ import ch.tutteli.atrium.reporting.reporter
 annotation class ExperimentalWithOptions
 
 /**
- * Uses the given [textRepresentation] as representation of the subject instead of the current representation
- * (which defaults to the subject itself).
+ * Wraps the given [textRepresentation] into a [RawString] and uses it as representation of the subject
+ * instead of the so far defined representation (which defaults to the subject as such).
+ *
+ * In case [Expect.maybeSubject] is not defined i.e. [None], then the previous representation is used.
  */
 @ExperimentalWithOptions
-fun <T> RootExpect<T>.withOptions(textRepresentation: String): Expect<T> =
-    withOptions(ExpectOptions(representation = RawString.create(textRepresentation)))
+fun <T> RootExpect<T>.withRepresentation(textRepresentation: String): Expect<T> =
+    withOptions { withRepresentation(textRepresentation) }
+
+/**
+ * Uses the given [representationProvider] to retrieve a representation which can be based on the current
+ * subject where it is used as new representation of the subject
```

Comment:
```suggestion
 * subject. The created value is used as the new representation of the subject
```

Reply:
I know it is quite a long sentence but I have the feeling the first sentence should cover the main functionality